honeybadger-java/src/main/java/io/honeybadger/reporter/HoneybadgerReporter.java

```diff
@@ -66,14 +66,12 @@ public HoneybadgerReporter(final ConfigContext config) {
         if (config.getHoneybadgerUrl() == null) {
             throw new IllegalArgumentException("Honeybadger URL must be set");
         }
-        if (config.getMaximumErrorReportingAttempts() == null) {
-            throw new IllegalArgumentException("Honeybadger property MaximumErrorReportingAttempts must be set");
+        if (config.getMaximumErrorReportingRetries() == null) {
+            throw new IllegalArgumentException("Honeybadger property getMaximumErrorReportingRetries must be set");
```

Comment:
I probably am not understanding this -- is the configurable property name really `getMaximumErrorReportingRetries`, or is it actually something like `MaximumErrorReportingRetries`?

Reply:
Yeah, it's MaximumErrorReportingRetries in the object itself. I decided for Principle of Least Surprise when a system admin is tearing their hair out to replace this with the SystemProperty name in my latest checkin.